api/resource.rb

```diff
@@ -45,6 +44,9 @@ module Properties
 
     include Properties
 
+    # Parameters can be overridden via Provider::PropertyOverride
+    attr_reader :parameters
```

Comment:
You will need to add some code in resource_overrides to enable this.

Reply:
I did already. Note that in `get_properties` I changed it to be `api_entity.all_user_properties`, which will include all parameters as well.

Reply:
Gotcha 👍